rest-api-spec/build.gradle

```diff
@@ -146,11 +146,9 @@ tasks.named("yamlRestCompatTest").configure {
     'explain/21_source_filtering_with_types/Source filtering',
     'explain/31_query_string_with_types/explain with query_string parameters',
     'explain/40_mix_typeless_typeful/Explain with typeless API on an index that has types',
-    'field_caps/10_basic/Get date_nanos field caps',
     'field_caps/30_filter/Field caps with index filter',
-    'get/100_mix_typeless_typeful/GET with typeless API on an index that has types',// failing due to include_type_name #48632
-    'get/21_stored_fields_with_types/Stored fields', // failing due to include_type_name #48632
-    'get/71_source_filtering_with_types/Source filtering',// failing due to include_type_name #48632
+    // WILL NOT BE FIXED - failing due to not recognising missing type (the type path param is ignored)
```

Comment:
Could you give more details as to why this won't be fixed?

Reply:
not sure how to phrase this better in a comment.
This test is using a typed GET api and is expecting to get 0 results because of unknown type
```
 - do:
      catch:  missing
      get:
          index:  index
          type:   some_random_type
          id:     1
...
 - match:   { found:      false}

```

We are ignoring the type in get api, because there is no information about the type in index.

Reply:
Thanks for the explanation! Just wanted to check I understood.